ui/app/clinical/common/services/treatmentService.js

```diff
@@ -125,6 +125,21 @@ angular.module('bahmni.clinical')
             return deferred.promise;
         };
 
+        var sendPrescriptionSMS = function (visitUuid) {
+            var deferred = $q.defer();
+            var locale = $translate.instant(appService.getAppDescriptor().getConfigValue("smsLanguage") || Bahmni.Clinical.Constants.smsLanguage);
```

Comment:
Not sure about using a property in config just to pick up translation key. Why dont we just hardcode the KEY over here? It will be still be picked from relevant message bundles. Also what happens if the key is not defined in the message bundle, pls use a default in the backend

Reply:
refactored it by hardcoding the KEY. If this is not defined, english will be picked up as default in the backend